packages/paste-website/src/pages/components/textarea/index.mdx

```diff
@@ -196,6 +196,18 @@ The textarea should include the base textarea, along with supporting elements to
 </>`}
 </LivePreview>
 
+### Resizeable Textarea
+
+By default, the textarea is not resizeable. To change this, add the prop `resize='vertical'`.
+
+<LivePreview scope={{Label, HelpText, TextArea}} language="jsx">
+  {`<>
```

Comment:
todo: missing id on the label and associated aria-labelledby

Reply:
This is not needed because the label has htmlFor

Reply:
This is not needed because the label has htmlFor